PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -76,8 +76,8 @@ def AND(tokens):
                 )
         ),
         Photon = cms.PSet(
-            id = cms.int32(22),
-            sel = cms.string("type(92) && pt > 20 && coll('hltEgammaCandidates')"),
+            id = cms.int32(18),
```

Comment:
Please do not change this ID. It is purely to distinguish objects in the TriggerObjects table in NanoAOD, and loosely follows the PDG IDs.